models/calendar.yml

```diff
@@ -288,10 +288,10 @@
 
     - 9:00 PM:
         type: Talk
-        title: TBD
-        emoji: 🤔
+        title: Push and Pull: Boids!
+        emoji: 🦅
         description:
```

Comment:
I believe you need a `|` to make this work, like so:
```yml
  description: | 
    Simple rules, surprising complexity! Some simple physics demos and the basics behind them.
```

Alternatively, you can put it on the same line:
```yml
  description: Simple rules, surprising complexity! Some simple physics demos and the basics behind them.
```